webpack.config.renderer.dev.dll.js

```diff
@@ -20,20 +18,163 @@ export default merge.smart(baseConfig, {
 
   devtool: 'eval',
 
-  mode: 'development',
-
   target: 'electron-renderer',
 
   externals: ['fsevents', 'crypto-browserify'],
 
   /**
    * Use `module` from `webpack.config.renderer.dev.js`
    */
-  module: require('./webpack.config.renderer.dev').module,
+  module: {
```

Comment:
Does this replace the bundling that would have occurred on code previously not treated as a webpack external?